packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.test.tsx

```diff
@@ -104,6 +105,17 @@ describe('<AdapterDayjs />', () => {
     });
   });
 
+  describe('customParseFormat plugin', () => {
+    it('should allow custom two year digit parsing', () => {
```

Comment:
Not sure you test anything here since the adapter has probably been added for another test before that one :grimacing:

Reply:
yep ... you are right ... gotta think of a better way 🤔